File: JavaScript/1043.js
var input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
var lines = input.split('\n');
const pegarValores = (line) => line.split(" ").map(a => +a);
let [a,b,c] = pegarValores(lines.shift())
let calc;
if(a<(b+c) && b<(a+c) && c<(a+b)){
    calc = a+b+c
    console.log(`Perimetro = ${calc.toFixed(1)}`)
}else{
    calc = ((a+b)*c)/2
    console.log(`Area = ${calc.toFixed(1)}`)
}

File: Python/1043.py
lista = (input().split(" "))
a,b,c = lista
a = float(a)
b = float(b)
c = float(c)
calc = 0
if(a<(b+c) and b<(a+c) and c<(a+b)):
    calc = a+b+c
    print('Perimetro = %.1f'%calc)
else:
    calc = ((a+b)*c)/2
    print('Area = %.1f'%calc)
